freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -70,11 +68,37 @@ def handle(self, event):
             self.log_info(msg)
             return []
 
-        # Get original nvrs of all builds in the advisory
-        original_nvrs = [build.original_nvr for build in db_event.builds]
+        # Get builds NVRs from the advisory attached to the message/event
+        affected_nvrs = set()
```

Comment:
Both `original_nvrs` and `affected_nvrs` are misleading, can we rename it to `shipped_nvrs`?